hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SharedNetworkLauncherSessionListener.java

```diff
@@ -94,6 +102,8 @@ public void testPlanExecutionStarted(@NonNull final TestPlan testPlan) {
             final boolean clprSystemPropertyEnabled =
                     Boolean.parseBoolean(System.getProperty("clpr.clprEnabled", "false"));
             final boolean enableClprOverrides = hasClprTests || clprSystemPropertyEnabled;
+            final boolean hasRestartTests = hasTaggedTestNode(testPlan, Set.of(TestTags.RESTART))
+                    || hasTestClasses(testPlan, QUIESCENCE_TEST_CLASSES);
```

Comment:
I don't love tracking these class names in the code itself. `hasTaggedTestNode(testPlan, Set.of(TestTags.RESTART))` should already include `QuiesceThenMixedOpsRestartTest`, and we can add a`@Tag(RESTART)` to `JustQuiesceTest` to achieve the same effect.